packages/type-storage/src/wellKnownKeys.ts

```diff
@@ -0,0 +1,55 @@
+import CodecArray from '@polkadot/api-codec/codec/Array';
+import String from '@polkadot/api-codec/String';
+
+import { createFunction } from './utils/createFunction';
+import { StorageFunctionModifier } from '@polkadot/api-codec/Metadata';
+
+interface WKKMetadata {
+  documentation: CodecArray<String>;
+  name: String;
+  type: any; // TODO
+}
+
+// Small helper function to factorize code on this page.
+const createWKKFunction = (prefix: String, functionMetadata: WKKMetadata) =>
+  // TODO The expected 2nd argument is a StorageFunctionMetadata, but we actually only need the fields `documentation`
+  // and `type` of it, so in order to not input other fields (byteLength, fromJSON...) we lazily cast as any.
+  createFunction(prefix, { ...functionMetadata, modifier: new StorageFunctionModifier().fromJSON(0) } as any, {
+    isUnhashed: true
+  });
+
+export const code = createWKKFunction(new String(':code'), {
+  documentation: new CodecArray(String, ['Wasm code of the runtime.']),
```

Comment:
In my JS tests this was an issue.... since we have String & String. In my case this defaulted to the JS version. All worked fine until I wanted to do stuff on the actual string :(

Need a new name for `String`, as it stands, everywhere I use it, it is now called `CodecString`

Reply:
Maybe we should call it `Text`? (Don't quite like word prefixes unless they serve a purpose, it becomes muddy... Single character prefixes, well, too Java-like maybe)

Reply:
And while we are at it, Array -> Vector?